### DISCUSSION TITLE
总是Request timed out

### DISCUSSION THREAD

**Discussion by leoatchina** [MuiseDestiny/zotero-gpt#126] [General]
和zotero gpt 相关的操作全部这样
![image](https://user-images.githubusercontent.com/4470821/235070021-10c7e6cc-bcf0-4569-8fd3-f520b7152c44.png)

1. api key我生成了， 确定没复制错， 是美国的朋友充值注册的
![image](https://user-images.githubusercontent.com/4470821/235069607-c7857dac-9438-47a8-885f-a14b82d19731.png)
2. 上面的secret key我在本插件里用过， 新生成的zotero key也用过，都不行
3.  zotero key显示never uesd 
4.  secret key 我可用于 微信机器人， 所以这个openapi账号没问题
![image](https://user-images.githubusercontent.com/4470821/235069976-746e255c-1dc6-4048-83a4-4383f45efb15.png)

**Comment by MuiseDestiny** [comment]
重复

**Comment by Liutao2023** [comment]
同样的问题，
![image](https://user-images.githubusercontent.com/132579596/236213478-e1f8207d-8cb1-4411-bb8b-bf7c0c131fbb.png)
